src/collective/cover/browser/templates/layoutedit.pt

```diff
@@ -5,82 +5,87 @@
-      </div>
-      </form>
-    </div>
-    <div class="navbar">
-      <a id="btn-export" class="btn btn-inverse" tal:condition="can_export_layout"><span i18n:translate="">Export layout</span></a>
-      <a id="btn-save" class="btn btn-success saved"><span i18n:translate="">Saved</span></a>
-    </div>
-  </tal:block>
-</metal:main>
-
-</body>
+    <body>
+        <metal:js fill-slot="javascript_head_slot">
+            <script type="text/javascript"
+                    tal:define="navroot context/@@plone_portal_state/navigation_root_url"
+                    tal:attributes="src string:${navroot}/++resource++collective.cover/jss.min.js">
+            </script>
+            <script type="text/javascript"
+                    tal:define="navroot context/@@plone_portal_state/navigation_root_url"
+                    tal:attributes="src string:${navroot}/++resource++collective.cover/layout_base.js">
+            </script>
+            <script type="text/javascript"
+                    tal:define="navroot context/@@plone_portal_state/navigation_root_url"
+                    tal:attributes="src string:${navroot}/++resource++collective.cover/layout_edit.js">
+            </script>
+            <script type="text/javascript">
+            jQuery(document).ready(function() {
+                if (typeof(plone) !== 'undefined') {
+                    jQuery(window).unload(plone.UnlockHandler.execute);
+                }
+            });
+            </script>
+        </metal:js>
+        <metal:styles fill-slot="style_slot">
+            <link rel="stylesheet" type="text/css" media="screen" href=""
+                  tal:define="navroot context/@@plone_portal_state/navigation_root_url"
+                  tal:attributes="href string:${navroot}/++resource++collective.cover/layout_edit.css" />
+        </metal:styles>
+        <metal:main fill-slot="main">
+            <tal:block tal:define="can_export_layout view/can_export_layout">
+                <div id="sidebar">
+                    <ul class="layout-actions">
+                        <li>
+                            <a id="btn-row" class="btn btn-small btn-inverse draggable"><i class="icon-white"></i><span i18n:translate="">Row</span></a>
+                        </li>
+                        <li>
+                            <a id="btn-column" class="btn btn-small btn-inverse draggable"><i class="icon-white"></i><span i18n:translate="">Column</span></a>
+                        </li>
+                    </ul>
+                    <div tal:replace="structure context/@@tile_list"></div>
+                </div>
+                <div class="layout" tal:attributes="data-layoutmanager-settings view/layoutmanager_settings">
+                    <div tal:define="layout nocall:context/@@layout"
+                         tal:replace="structure layout/render_layout_edit" />
+                    <div id="resizer" title="Resize Column">
```

Comment:
Moved resizer div from JS to this template (as it must have one copy of it in the entire template)